module3-crud-and-data-models/r1-schema-and-data-models/README.md

```diff
@@ -0,0 +1,219 @@
+## Introduction To Database Schemas
+
+A database schema is a blueprint or architecture of how our data will look. It doesn’t hold data itself, but instead describes the shape of the data and how it might relate to other tables or models.
+
+For example, this is a `customer` schema in MongoDB using Mongoose.
+
+```js
+const mongoose = require("mongoose");
+
+const customer = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  zipcode: {
+    type: Number,
+  },
+});
+
+module.exports = mongoose.model("customer", customer);
+```
+
+The same `customer` schema in PostgreSQL database using sequelize.
+
+```js
+const Sequelize = require("sequelize");
+
+module.exports = (sequelize, DataTypes) => {
+  const Customer = sequelize.define("customer", {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    zipcode: {
+      type: DataTypes.INTEGER,
+    },
+  });
+  return Customer;
+};
+```
+
+We can see that the schema definition for the `customer` has a `name` that is type `string` and a `zipcode` that is of type `number`.
+
+## Introduction to CRUD operations
```

Comment:
In the previous section, I think we will need some explanation of what a "model" is. Otherwise we are mentioning this word in this new section without students being properly introduced to its meaning.

To be specific we can explain similar to how [this Mongoose article](https://mongoosejs.com/docs/models.html) does, given that all labs will be in Mongoose for this module.